src/tests/Microsoft.PowerFx.Interpreter.Tests.Shared/RuntimeErrorTests.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System.Globalization;
+using System.Linq;
+using Microsoft.PowerFx.Types;
+using Xunit;
+
+namespace Microsoft.PowerFx.Core.Tests
+{
+    public class RuntimeErrorTests
```

Comment:
>public [](http://example.com/codeflow?start=4&length=6)

Note that error messages can also be retrieved programmatically:

IfError(""&1/0, FirstError.Message) // "Invalid operation: division by zero." // String


ToString/Parse expressions also depend on runtime culture
Value("123.456") // parses in current culture.